src/App.test.js

```diff
@@ -1,8 +1,11 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render } from '@testing-library/react';
+import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
+// import TestRenderer from 'react-test-renderer';
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  expect(render(<Calculator />)).toMatchSnapshot();
+  expect(render(<Home />)).toMatchSnapshot();
+  expect(render(<Quotes />)).toMatchSnapshot();
```

Comment:
Nice job using jest snapshots to test your components✨. However, according to the project requirements, you are also supposed to write unit tests for **all** your components using the react testing library. The react testing library provides many useful functions for interacting with your components during testing. Take a look at this [tutorial](https://testing-library.com/docs/react-testing-library/example-intro) to see examples of the library in action. You can also have a look at this [cheatsheet](https://testing-library.com/docs/react-testing-library/cheatsheet) to see common queries used in the library. 

Please create separate test files for each of your components using functions from the react testing library🙏